webpack.config.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   output: {
     filename: 'main.js',
     path: path.resolve(buildFolder, 'js')
-  }
+  },
+  mode: 'production'
```

Comment:
How do we debug development code if the output is always set to production? Is it always minified?

Reply:
solved